pallets/messages/src/types.rs

```diff
@@ -31,20 +31,35 @@ impl<MaxDataSize> Message<MaxDataSize>
 where
 	MaxDataSize: Get<u32> + Clone,
 {
-	/// Helper function to handle response type [`MessageResponse`] for RPC.
+	/// Helper function to handle response type [`MessageResponse`] depending on the Payload Location (on chain or IPFS)
 	pub fn map_to_response<BlockNumber>(
 		&self,
 		block_number: BlockNumber,
 		payload: Vec<u8>,
```

Comment:
This was in the original code, but I think `payload` could be referenced from `self.payload` instead of needing to be passed in.